world/races.py

```diff
@@ -0,0 +1,330 @@
+    """
+    def __init__(self, name, desc, bonuses):
+        self.name = name.capitalize()
+        self._desc = None
+        self.desc = desc
+        self.bonuses = bonuses
+
+    @property
+    def desc(self):
+        """Returns a formatted description of the Focus.
+
+        Note:
+            The setter for this property only modifies the content
+            of the first paragraph of what is returned.
+        """
+        desc = "{{b{}{{n\n".format(self.name)
+        desc += fill(self._desc)
+        desc += "\n\n"
+        desc += fill("Adventurers with the {{b{}{{n focus gain {}.".format(
+                    self.name,
+                    _format_bonuses(self.bonuses)
+                ))
+        desc += "\n\n"
+        return desc
+
+    @desc.setter
+    def desc(self, value):
+        self._desc = value
+
+
+class Human(Race):
+    """Class representing human attributes."""
+    def __init__(self):
+        super(Human, self).__init__()
+        self.name = "Human"
+        self.plural = "Humans"
+        self.size = "medium"
+        self.desc = ("Humans are the most wide-spread of all the races. The "
+                     "human traits of curiosity, resourcefulness and "
+                     "unyielding courage have helped them to adapt, survive "
+                     "and prosper in every world they have explored.")
+        self.foci = [load_focus(f) for f
+                     in ('agility', 'cunning', 'prestige')]
+        self.bonuses = {'WILL': 1}  # +3 languages
+
+
+class Elf(Race):
+    """Class representing elf attributes."""
+    def __init__(self):
+        super(Elf, self).__init__()
+        self.name = "Elf"
+        self.plural = "Elves"
+        self.size = "medium"
+        self.desc = ("Elves are graceful, slender demi-humans with delicate "
+                     "features and pointy ears. Elves are known to use magic "
+                     "spells, but prefer to spend their time feasting and "
+                     "frolicking in wooded glades. They rarely visit cities of "
+                     "men. Elves are fascinated by magic and never grow weary "
+                     "of collecting spells or magic items. Elves love "
+                     "beautifully crafted items and choose to live an agrarian "
+                     "life in accord with nature. ")
+        self.foci = [load_focus(f) for f
+                     in ('agility', 'spirit', 'alertness')]
+        self.bonuses = {}
+
+
+class Dwarf(Race):
+    """Class representing dwarf attributes."""
+    def __init__(self):
+        super(Dwarf, self).__init__()
+        self.name = "Dwarf"
+        self.plural = "Dwarves"
+        self.size = "small"
+        self.desc = ("Dwarves are short, stocky demi-humans with long, "
+                     "respectable beards and heavy stout bodies. Their skin "
+                     "is earthen tone and their hair black, gray or dark "
```

Comment:
Apologies that my only line comments were grammar policing! :) Great work on all the tests, thanks @feend78 .

What are your thoughts on slots/limbs with this PR? Do we wait for a refactor of eq to develop that or is the intent to decrease available slots?